packages/grant-explorer/src/features/common/PassportBanner.tsx

```diff
@@ -43,6 +43,12 @@ export default function PassportBanner(props: {chainId?: string, roundId?: strin
         const res = await fetchPassport(address, PASSPORT_COMMUNITY_ID);
         if (res.ok) {
           const json = await res.json();
+
+          if (json.status == "PROCESSING") {
+            await callFetchPassport();
```

Comment:
won't this result in a possibly infinite loop? If it never changes from processing. Also, we should probably have exponential backoff here. I'll merge anyway, but I'll also add a TODO here to address this.